src/virtualship/cli/commands.py

```diff
@@ -1,9 +1,11 @@
+from datetime import datetime
 from pathlib import Path
 
 import click
+import copernicusmarine
```

Comment:
Does this then become a dependency in the pyproject.toml and the conda-forge recipe too?

Reply:
and as a dependency in `environment.yml`

Reply:
Done in 391f5ef161a303c6e0e98f2d6fc6f0d3bc05950c